samples/web/asp-mvc-application/Core_6/AsyncPagesMVC/Global.asax.cs

```diff
@@ -39,10 +39,12 @@ protected void Application_Start()
 
         // Register the MVC controllers.
         builder.RegisterControllers(typeof(MvcApplication).Assembly);
-
-        // Set the dependency resolver to be Autofac.
+        // Register the endpoint as a factory as the instance isn't created yet.
+        builder.Register(ctx => endpoint);
```

Comment:
Is this then resolved as a singleton/singleinstance? I'm not familiar enough with Autofac. @Particular/container-maintainers